README.md

```diff
@@ -27,10 +37,13 @@ You'll also need to add the following line to your `/etc/hosts` file:
 ### Running the app
 
 ```
-bin/start-dependencies
+If not using Apple chipset use: 'bin/start-dependencies'
```

Comment:
_not a blocker_
Is this because of the dockerize image to check for "is it up yet"? Otherwise it's using compose as well